src/data/faq_de.json

```diff
@@ -325,6 +325,14 @@
                         "anchor": "minimum_requirements",
                         "active": true,
                         "textblock": [
+                            "<b>AKTUELL 8. Februar 2021</b><br/>Seit dem 5. Februar 2021 werden nur die Corona-Warn-App (CWA) Versionen 1.5 und höher unterstützt. Aus diesem Grund erhalten Nutzer mit älteren Versionen einen Hinweis, Ihre App entsprechend zu aktualisieren. Dies gilt sowohl für iOS als auch Android. Parallel gelten jedoch bestimmte Mindestvoraussetzungen an das Betriebssystem: Android mindestens 6.0 und iOS mindestens 13.7.",
+                            "iOS Nutzer, die bisher bspw. iOS 13.5 genutzt haben, müssen Ihr iOS Betriebsystem nun zwingend auf mindestens iOS 13.7 aktualisieren, um die CWA weiterhin zu nutzen. Nur in diesem Fall kann eine technisch korrekte Funktion der App sichergestellt werden.",       

```

Comment:
Ihr iOS Betriebsystem => ihr iOS Betriebssystem (Grammar and Typo)

Reply:
ok